charts/live/Chart.yaml

```diff
@@ -4,7 +4,7 @@ description: CodeTogether Live provides pair programming and collaborative codin
 
 type: application
 version: 1.4.20
-appVersion: "2024.2.0"
```

Comment:
I think that we should keep 2024.2.0 here and update the version above (1.4.20) instead.

Reply:
Just changed version to 1.4.21